app/presenters/tree_builder_policy.rb

```diff
@@ -1,6 +1,4 @@
 class TreeBuilderPolicy < TreeBuilder
-  has_kids_for MiqPolicy, [:x_get_tree_po_kids]
```

Comment:
Is this visually redundant?

Reply:
@skateman yes, trees in the new explorers do not show child nodes, these trees will be gone in follow up PRs anyway. If i leave the child nodes in the trees then i will have to focus on making code changes to get the links working to jump from one explorer to another and i don't see any point in investing time getting that working as explorers/trees will be gone soon.